src/models/relationship.js

```diff
@@ -21,8 +22,14 @@ import {camelToSnake, snakeToCamel} from '../util';
 
 export default function relationship(bookshelf) {
 	const Relationship = bookshelf.Model.extend({
+		date() {
+			return this.belongsTo('RelationshipDate', 'id');
+		},
 		format: camelToSnake,
 		idAttribute: 'id',
+		ordinal() {
+			return this.belongsTo('RelationshipAttributeOrdinal', 'id');
+		},
```

Comment:
I've been doing some more research into database designs, and comparing how the MusicBrainz database schema works regarding relationship attributes.

I'm not yet at a point where I have a clear idea and a path forward, and I need to think about it some more before being sure that we can do everything we want this way (with a RelationshipAttributeOrdinal)

Sorry about the wait !
If you want to have a look with me, I'm looking at the [schema docs here](https://musicbrainz.org/doc/MusicBrainz_Database/Schema#Relationship_table_structure) as well as inspecting a sample MB database locally to try to understand how all the pieces fit together.


One crucial piece I'm currently missing are the tables that will allow us to describe which relationship_type will have which relationship attributes (for example, how do we describe that the "Work is part of Series" needs an ordinal attribute for ordering)